ProMotion.gemspec

```diff
@@ -25,6 +25,9 @@ Gem::Specification.new do |gem|
   gem.version       = ProMotion::VERSION
 
   gem.add_runtime_dependency("methadone", "~> 1.7")
+  gem.add_runtime_dependency("motion_print", "~> 1.0")
+
+  gem.add_development_dependency("motion_print", "~> 1.0")
```

Comment:
You can just do `gem.add_dependency`. I don't remember why we need it as a development dependency, tbh.